src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  margin: 8px;
+}
+
+.search-form {
+  margin: 20px auto 0;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 70px;
+}
+
+.search-input {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  background-image: url(./images/icon-search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  background-size: 19px 19px;
+  padding-left: 62px;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document